packages/app-market/src/components/settings/settings-installed.tsx

```diff
@@ -1,8 +1,278 @@
+    openModal: () => void,
+    installationId?: string,
+    appId?: string,
+  ) =>
+  () => {
+    if (installationId && appId) {
+      setInstallationDetails({ installationId, appId })
+      openModal()
+    }
+  }
+
+export const getAppIds = (installations: InstallationModelPagedResult | null) => () =>
+  installations?.data?.map((installation) => installation.appId).filter(isTruthy)
 
 export const SettingsInstalled: FC = () => {
-  return <Title>Installations</Title>
+  const [installationDetails, setInstallationDetails] = useState<null | InstallationDetails>(null)
+  const [pageNumber, setPageNumber] = useState<number>(1)
+  const { connectSession } = useReapitConnect(reapitConnectBrowserSession)
+  const { Modal, openModal, closeModal } = useModal()
+  const clientId = connectSession?.loginIdentity.clientId
+  const email = connectSession?.loginIdentity.email ?? ''
+
+  const [installations, installationsLoading, , refetchInstallations] = useReapitGet<InstallationModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getInstallations],
+    queryParams: {
+      pageNumber,
+      pageSize: 12,
+      clientId,
+    },
+    fetchWhenTrue: [clientId],
+  })
+
+  const [, , uninstallApp, uninstallSuccess] = useReapitUpdate<TerminateInstallationModel, null>({
+    reapitConnectBrowserSession,
+    action: updateActions(window.reapit.config.appEnv)[UpdateActionNames.terminateInstallation],
+    uriParams: {
+      installationId: installationDetails?.installationId,
+    },
+  })
+
+  const appIds = useMemo(getAppIds(installations), [installations])
+
+  const [apps] = useReapitGet<AppSummaryModelPagedResult>({
+    reapitConnectBrowserSession,
+    action: getActions(window.reapit.config.appEnv)[GetActionNames.getApps],
+    queryParams: {
+      id: [...new Set(appIds)],
+    },
+    fetchWhenTrue: [appIds],
+  })
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<TerminateInstallationModel>({
+    resolver: yupResolver(uninstallAppSchema),
+    defaultValues: {
+      terminatedBy: '',
+      terminatedReason: '',
+      terminatesOn: new Date().toISOString(),
+    },
+  })
+
+  useEffect(handleUninstallSuccess(refetchInstallations, closeModal, uninstallSuccess), [uninstallSuccess])
+
+  return (
+    <>
+      <Title>Installations</Title>
+      {installationsLoading ? (
+        <Loader />
+      ) : installations?.totalCount ? (
+        <>
+          <BodyText hasGreyText hasSectionMargin>
+            The table below gives you the information about Reapit AppMarket apps and integrations you have installed
+            currently or previously. In addition, you can uninstall apps for all users of your organisation by using the
+            call to action on each row. For more information on the installations table
+            <a onClick={openNewPage('')}> see here</a>.
```

Comment:
suppose to be empty?

Reply:
For now, no docs as yet so nowhere to link to